Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
 
+gem "activerecord", ">=7"
```

Comment:
This isn't required if it's in the gemspec.

Reply:
Answered [here](https://github.com/clio/jit_preloader/pull/64#issuecomment-2070031424)